aiowebostv/webos_client.py

```diff
@@ -675,18 +681,21 @@ async def request(
             res = self.futures[uid]
         try:
             await self.command(cmd_type, uri, payload, uid)
+            async with asyncio.timeout(RECEIVE_TIMEOUT):
```

Comment:
I'd name this something other than `RECEIVE_TIMEOUT` even if its simply a new constant that equals the current one to better show the intent is to also handle the timeout where we wait for drain in case the protocol is paused. Alternatively a comment here to explain